tests/ui/async-await/future-sizes/async-awaiting-fut.stdout

```diff
@@ -2,38 +2,34 @@ print-type-size type: `[async fn body@$DIR/async-awaiting-fut.rs:21:21: 24:2]`:
 print-type-size     discriminant: 1 bytes
 print-type-size     variant `Unresumed`: 0 bytes
 print-type-size     variant `Suspend0`: 3077 bytes
-print-type-size         local `.__awaitee`: 3077 bytes, offset: 0 bytes, alignment: 1 bytes
+print-type-size         local `.__awaitee`: 3077 bytes
 print-type-size     variant `Returned`: 0 bytes
 print-type-size     variant `Panicked`: 0 bytes
 print-type-size type: `[async fn body@$DIR/async-awaiting-fut.rs:10:64: 19:2]`: 3077 bytes, alignment: 1 bytes
 print-type-size     discriminant: 1 bytes
-print-type-size     variant `Unresumed`: 2051 bytes
-print-type-size         padding: 1026 bytes
```

Comment:
Hrm, not sure how to judge that one. It's a variant in an explicit-discriminant enum and I didn't pay much attention to those since they don't benefit from niche optimization. At least in this particular case it doesn't affect the overall enum size.

I also don't understand the meaning of `, offset: 0 bytes, alignment: 1 bytes` disappearing on other variants.

Should I investigate this?

Reply:
This is a big improvement for the variant, so nothing to investigate. Although yeah, it doesn't matter at all (and could even just be print-type-size variant size calculation being odd).